ExampleApp/ExampleApp/AudioPlayerView.m

```diff
@@ -254,6 +271,27 @@ -(void) muteButtonPressed
 	}
 }
 
+-(void) rateControlChanged
+{
+    switch (rateControl.selectedSegmentIndex) {
```

Comment:
Can you put the brace on the next line just to match the rest of the coding style ;-)